server/serviceClients/authClient/local.js

```diff
@@ -0,0 +1,83 @@
+// This is an example of the simplest scenario when the widget is hosted and used in the same secure domain
+// where as the provider credentials are safe to be stored in the widget server, hence as soon as the auth token is valid
+// Or no auth token needed at all, 
+// simply return the provider credentials, coppied from : 
+// const providerConfig = require('../../../../example/application/configuration')
+
+const {decrypt} = require('../../utils');
+const config = require('../../config');
+
+export class AuthClient {
+  token;
+  constructor(token){
+    this.token = token;
+  }
+
+  async getSecretExchange(iv){
```

Comment:
This function needs a unit test

Reply:
The goal for this repo is to have complete test coverage to ensure confidence whenever we make a change. We're a long way from this goal but it would be really helpful to start by making sure at least the new code that's added is covered by tests. Wes has been working on adding some tests to provide a good example which we can follow when writing new tests. Have those been merged yet @wesrisenmay-mx? Can you share a link to one?

Reply:
Unit/Integration test example: https://github.com/Universal-Connect-Project/ucw-app/blob/main/server/providers/mx.test.ts

End to end test examples:
https://github.com/Universal-Connect-Project/ucw-app/blob/main/cypress/e2e/mx.cy.ts

https://github.com/Universal-Connect-Project/ucw-app/blob/main/cypress/e2e/sophtron.cy.ts